### DISCUSSION TITLE
Best model only for language detection

### DISCUSSION THREAD

**Discussion by ddorian** [SYSTRAN/faster-whisper#739] [Q&A] [answered]
Hi,

What would you think is the best (fastest with enough accuracy) model only for language detection? 

I searched around but didn't find any conclusive answer.

**Comment by stri8ed** [comment, marked as answer]
Enough accuracy is a subjective metric, So I wouldn't expect to find a conclusive answer. If you have access to a good set of multilingual training data, You can create some benchmarks by sampling a small clip from each, and get a better sense of the performance.

**Comment by shaunck96** [comment]


**Reply by ddorian** [reply]
Assuming you mean https://pypi.org/project/langdetect/, it's for text & it's old.
